recipe/meta.yaml

```
@@ -18,12 +20,14 @@ source:
     - patches/do_not_vendor_packmol.patch
     - patches/remove_mtune_native_for_m1.patch  # [arm64]
     - patches/remove_nab2c_and_friends.patch  # [arm64]
+    - patches/0001-do-not-clean-compiler-flags.patch
 
 build:
-  number: 6
-  skip: True  # [win or py2k or ppc64le or aarch64]
+  number: {{ build }}
+  skip: True  # [win or py2k or ppc64le or aarch64 or py!=39]
```

Revert after everything is fine:
```suggestion
  skip: True  # [win or py2k or ppc64le or aarch64]
```